src/app/app.module.ts

```diff
@@ -355,45 +355,36 @@ const monacoConfig: NgxMonacoEditorConfig = {
         BlockComparisonBuilderBoxComponent,
         LinkActionComponent
     ],
-  imports: [
-    FormlyModule.forRoot({}),
-    BrowserModule,
-    AppRoutingModule,
-    BrowserAnimationsModule,
-    AppMaterialModule,
-    ServiceWorkerModule.register('ngsw-worker.js', {enabled: environment.production}),
-    TranslateModule.forRoot({
-      loader: {
-        provide: TranslateLoader,
-        useFactory: HttpLoaderFactory,
-        deps: [HttpClient]
-      }
-    }),
-    HttpClientModule,
-    ReactiveFormsModule,
-    FormsModule,
-    NgxTaggerModule,
-    MessagesModule,
-    DragDropModule,
-    LeafletModule,
-    LeafletMarkerClusterModule,
-    FormlyModule.forRoot(config),
-    FormlyMaterialModule,
-    MonacoEditorModule.forRoot(monacoConfig),
-    MatAutocompleteModule,
-    AgGridModule,
-    MatChipsModule
-  ],
+    bootstrap: [AppComponent], 
+    imports: [
+        FormlyModule.forRoot({}),
+        BrowserModule,
+        AppRoutingModule,
+        BrowserAnimationsModule,
+        AppMaterialModule,
+        ServiceWorkerModule.register('ngsw-worker.js', { enabled: environment.production }),
+        TranslateModule.forRoot({
+            loader: {
+                provide: TranslateLoader,
+                useFactory: HttpLoaderFactory,
+                deps: [HttpClient]
+            }
+        }),
+        ReactiveFormsModule,
+        FormsModule,
+        NgxTaggerModule,
+        MessagesModule,
+        DragDropModule,
+        LeafletModule,
+        LeafletMarkerClusterModule,
+        FormlyModule.forRoot(config),
+        FormlyMaterialModule,
+        MonacoEditorModule.forRoot(monacoConfig),
+        MatAutocompleteModule,
+        AgGridModule,
+        MatChipsModule
+    ], 
     providers: [
-        // This service is from old legacy code and no longer used,
-        // so I'm commenting out the init() function and the whole thing
-        // can be removed once confirmed nothing is still using it.
-        // {
-        //   provide: APP_INITIALIZER,
-        //   multi: true,
-        //   useFactory: (schemaRepo: SchemaRepositoryService) => () => schemaRepo.init(),
-        //   deps: [SchemaRepositoryService]
-        // },
```

Comment:
Good point. Is used in a confusing way. I believe it can be deleted but probably is better to not add more complications to  this PR so I add it back.